extensions/prometheus-grafana-k8s/README.md

```diff
@@ -0,0 +1,98 @@
+# prometheus-grafana Extension
+
+
+This is the prometheus-grafana extension.  Add this extension to your json file as shown below to automatically enable prometheus and grafana in your new Kubernetes cluster.

```

Comment:
Could you kindly replace "your json file" with "the api model you pass as input into `acs-engine`"